.circleci/config.yml

```diff
@@ -132,8 +139,7 @@ jobs:
                 "commit message"
               PATHS=sql
             else
-              echo "Check dry run for modified queries"
-              PATHS="$(git diff origin/main... --name-only --diff-filter=d -- sql)"
+              PATHS="sql/bigquery-etl-integration-test"
```

Comment:
It would probably take me a while to figure out where does `sql/bigquery-etl-integration-test` come from if I was reading this couple of months from now. Maybe it's worth adding a comment that this path got created in `bqetl stage deploy`?